src/Microsoft.Performance.Toolkit.Engine/EngineExecutionResults.cs

```diff
@@ -215,5 +223,217 @@ public bool TryQueryOutput<T>(DataOutputPath dataOutputPath, out T data)
             data = default(T);
             return false;
         }
+
+        /// <summary>
+        ///     Checks if the given <paramref name="tableDescriptor"/> has data available.
+        /// </summary>
+        /// <param name="tableDescriptor">
+        ///     The table to check.
+        /// </param>
+        /// <returns>
+        ///     <c>true</c> the table has data available;
+        ///     <c>false</c> otherwise.
+        ///     <c>null</c> the table does not support checking if data is available, 
+        ///     recommend to call <see cref="BuildTable(TableDescriptor)"/>.
```

Comment:
what does this mean

Reply:
IsDataAvailable is not required to be implemented. 

Should reword a little?

Reply:
the part I was confused on is "recommend to call <see..."

Reply:
Ah, wanted to say that the caller should call BuildTable directly without needing to check if it has data